tomviz/EmdFormat.cxx

```diff
@@ -113,6 +113,33 @@ class EmdFormat::Private
                      H5T_STD_U32LE);
   }
 
+  bool attribute(const std::string& group, const std::string& name,
+                 std::string& value)
+  {
+    if (H5Aexists_by_name(fileId, group.c_str(), name.c_str(), H5P_DEFAULT) <=
+        0) {
+      // The specified attribute does not exist.
+      cout << group << name << " not found!" << endl;
+      return false;
+    }
+
+    hid_t attr = H5Aopen_by_name(fileId, group.c_str(), name.c_str(),
+                                 H5P_DEFAULT, H5P_DEFAULT);
+    hid_t type = H5Aget_type(attr);
+    char* tmpString;
+    if (H5Aread(attr, type, &tmpString) < 0) {
+      cout << "Failed to read attribute " << group << " " << name <<endl;
```

Comment:
The endl needs a space in front, causing a clang-format warning.

Reply:
I've squashed in this style fix.